Mathlib/Analysis/Analytic/Basic.lean

```diff
@@ -617,6 +617,10 @@ theorem AnalyticOn.add {s : Set E} (hf : AnalyticOn 𝕜 f s) (hg : AnalyticOn 
   fun z hz => (hf z hz).add (hg z hz)
 #align analytic_on.add AnalyticOn.add
 
+theorem AnalyticOn.neg {s : Set E} (hf : AnalyticOn 𝕜 f s) :
+    AnalyticOn 𝕜 (-f) s :=
+  fun z hz => (hf z hz).neg
```

Comment:
Can you put the conclusion on the same line as the hypotheses, and use "\maps" instead of "=>"? Thanks!